src/executor/validate/confirm.rs

```diff
@@ -0,0 +1,89 @@
+use {
```

Comment:
I didn't know that `use { ... }` is also possible. Wow, we can use `use { ... }` very well when we need to apply `#[cfg(...)]` to multiple imports. thanks!

Reply:
Yup, I quite like the syntax of `use { ... }`! :-)